packages/arb-token-bridge-ui/src/components/TransferPanel/useTransferReadiness.ts

```diff
@@ -182,7 +185,7 @@ export function useTransferReadiness({
   }, [nativeCurrency, erc20L1Balances])
 
   return useMemo(() => {
-    if (!amount) {
+    if (!Number(amount)) {
```

Comment:
NIT: Since amount is a string, this should check for isNan(Number(amount)) + checking for zero

Reply:
done